data/devices/xiaomi/04_redmi_note.yml

```diff
@@ -1,4 +1,36 @@
 ---
+- name: Xiaomi Redmi note 5
+  id: whyred
+
+  codenames:
+    - whyred
+    - Redmi Note 5
+
+  architecture: Kryo 260
```

Comment:
The `architecture` field should be `armeabi-v7a` (if 32-bit) or `arm64-v8a` (if 64-bit).